crypto/evp/p_lib.c

```diff
@@ -205,10 +205,14 @@ EVP_PKEY *EVP_PKEY_new(void)
     return ret;
 }
 
-void EVP_PKEY_up_ref(EVP_PKEY *pkey)
+int EVP_PKEY_up_ref(EVP_PKEY *pkey)
 {
     int i;
     CRYPTO_atomic_add(&pkey->references, 1, &i, pkey->lock);
```

Comment:
Done only on already modified code.
It's deserves a larger and separated review/commit to fix them all.